src/node/mempool_persist_args.cpp

```diff
@@ -21,3 +21,8 @@ fs::path MempoolPath(const ArgsManager& argsman)
 }
 
 } // namespace node
+
+fs::path MemPoolMinFeePath(const ArgsManager& argsman)
+{
+    return argsman.GetDataDirNet() / "mempool_min_fee.dat";
```

Comment:
Why add this as a separate file, rather than adding it to the end of the fee estimates data file which is already updated periodically and ignored if out of date?

Adding a method to the mempool to set the min fee based on that seems plausible? (Perhaps make it conditional on the new value being higher than the current minfee, and the mempool being less than 95% full?)

Reply:
I guess as long as it's only set shortly after startup (while the mempool is still loading?), it's probably fine?

Reply:
I updated the commits, it's now flushing to fee_estimates.dat